src/os/resourcesdir.zig

```diff
@@ -21,7 +21,10 @@ pub fn resourcesDir(alloc: std.mem.Allocator) !?[]const u8 {
 
     // This is the sentinel value we look for in the path to know
     // we've found the resources directory.
-    const sentinel = "terminfo/ghostty.termcap";
+    const sentinel = switch (comptime builtin.target.os.tag) {
+        .windows => "terminfo/ghostty.terminfo",
+        else => "terminfo/x/xterm-ghostty",
```

Comment:
This sentinel is incorrect and causes a complete failure to find the directory, on macOS at least-- that `x` is not right. Perhaps we should just put a dedicated sentinel file in the directory.